community/detector/cves/CVE-2021-29441/README.md

```diff
@@ -0,0 +1,19 @@
+# CVE-2021-29441 VulnDetector
```

Comment:
Can you move all the code into `community/detectors` rather than `community/detector`? Thanks!

Reply:
ok